plugins/katello/3.18/user_guide/simple_content_access/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Simple Content Access
+---
+
+# Simple Content Access
+
+This is a feature to simplify subscription management.  When an organization has Simple Content Access enabled, content hosts don't need to subscribe to a product in order to access product content.
```

Comment:
At the end of this sentence something like 'Instead, content hosts will automatically see repositories published into their selected Content View and Lifecycle Environment'